arcade/experimental/video_player.py

```diff
@@ -51,5 +60,20 @@ def get_video_size(self):
         return width, height
 
 
-window = VideoPlayer()
-arcade.run()
+class VideoPlayerView(arcade.View):
+
+    def __init__(self, path) -> None:
+        super().__init__()
+
+        self.video_player = VideoPlayer(path)
+
+    def on_draw(self):
+        self.clear()
+
+        self.video_player.draw(self.window.ctx, self.window.width, self.window.height)
```

Comment:
Maybe store the context in the video player instead. Make `width`, `height` and `size` property for both player types and support a `size: Tuple[int, int]` parameter for both draw methods? It can be optional using the view size if not supplied. Later we can add support for size in the cv2 video shader. No stress about that now.

Reply:
Updated https://github.com/pythonarcade/arcade/pull/1663/commits/a9f5133090d122f77571a90f14558a427a1d8d5a